### PR TITLE
use https links

### PR DESCRIPTION
**Pull request by bmwiedemann**
everywhere we can
and fix houndci URL

### CODE REVIEW

**Comment by rsalevsky** [comment]
Please don't change files in the vendor folder. Most of the files come directly from upstream.

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-The code and documentation is distributed under the [Apache 2 license](http://www.apache.org/licenses/LICENSE-2.0.html).
+The code and documentation is distributed under the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0.html).
 Contributions back to the source are encouraged.
 
-The [Crowbar Framework](https://github.com/crowbar/crowbar) is currently maintained by [SUSE](http://www.suse.com/) as
-an [OpenStack](http://openstack.org) installation framework but is prepared to be a much broader function tool. It was
-originally developed by the [Dell CloudEdge Solutions Team](http://dell.com/openstack).
+The [Crowbar Framework](https://github.com/crowbar/crowbar) is currently maintained by [SUSE](https://www.suse.com/) as
+an [OpenStack](https://www.openstack.org) installation framework but is prepared to be a much broader function tool. It was
+originally developed by the [Dell CloudEdge Solutions Team](https://dell.com/openstack).
```
Comment (aplanas) [inline README.md]: the dell certificate is not valid, I think that is better to use `http` here

File: crowbar_framework/vendor/assets/stylesheets/jquery/ui.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*! jQuery UI - v1.10.3 - 2013-11-25
-* http://jqueryui.com
+* https://jqueryui.com
```
Comment (aplanas) [inline crowbar_framework/vendor/assets/stylesheets/jquery/ui.scss]: as @rsalevsky this and more in `/vendor/` should not be altered

**Comment by tboerger** [comment]
:+1: but gating fails.

**Comment by bmwiedemann** [comment]
dropped the binary file from changes. should not have been in there anyway.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Crowbar: Core
 
-The code and documentation is distributed under the [Apache 2 license](http://www.apache.org/licenses/LICENSE-2.0.html).
+The code and documentation is distributed under the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0.html).
```
Comment (dirkmueller) [inline README.md]: please keep the license at http (see url)

File: crowbar_framework/barclamp_model/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Welcome to a Barclamp for the Crowbar Framework project
 =======================================================
 
-The code and documentation is distributed under the [Apache 2 license](http://www.apache.org/licenses/LICENSE-2.0.html).
+The code and documentation is distributed under the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0.html).
```
Comment (dirkmueller) [inline crowbar_framework/barclamp_model/README.md]: please keep it at http://

**Comment by sayalilunkad** [comment]
Some more files where you could replace http:
/chef/data_bags/crowbar/template-horizon.json: "help_url": "http://docs.openstack.org/",
/chef/cookbooks/postgresql/README.md: (some more similar ones in this file) 
 ["Omnibus" full stack installer (http://opscode.com/chef/install)
/chef/cookbooks/postgresql/recipes/config_pgtune.rb:  # http://rhaas.blogspot.com/2012/03/tuning-sharedbuffers-and-walbuffers.html

**Pull request merged**